src/components/02-components/data-table/data-table-experiences.tsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import {
+  Typography,
+  useTheme,
+} from '@mui/material';
+import OpenInBrowserIcon from '@mui/icons-material/OpenInBrowser';
+import EmailIcon from '@mui/icons-material/Email';
+import {
+  DataGrid,
+  GridColDef,
+  GridToolbarContainer,
+  GridToolbarExport,
+  GridToolbarColumnsButton,
+  GridToolbarFilterButton,
+  GridToolbarDensitySelector,
+  GridRowId,
+  GridActionsCellItem,
+} from '@mui/x-data-grid';
+
+export type Row = {
+  id: GridRowId;
+  col1: string;
```

Comment:
We need the ability to make rows optional. They are passed from the FE based on the GraphQL query and in some cases we need to do some tricks to pass Typescript checks: https://github.com/fourkitchens/s4-app-partners/actions/runs/5955540322/job/16154389142?pr=16
```suggestion
  col1?: string;
```